orchagent/vnetorch.cpp

```diff
@@ -2088,6 +2123,14 @@ void VNetRouteOrch::postRouteState(const string& vnet, IpPrefix& ipPrefix, NextH
             removeRouteAdvertisement(prefix_to_use);
         }
     }
+    if (route_state == "active")
```

Comment:
Should this code be in the if block at line#2115?

Reply:
The if block at LINE#2115 checks if the vnet is allowed to advertise prefix. The decap term should be added even if the  vnet is not allowed to advertise prefix.